### ISSUE TITLE
Fix documentation for attention mask shape

### ISSUE THREAD

**Issue by jotsif** [pytorch/pytorch#20851]
## 📚 Documentation

The attention mask shape should be `(L, S)` since it is added to the attention weights which is of shape (N, L, S). This is only in master, and not on pytorch.org yet.

https://github.com/pytorch/pytorch/pull/20850

**Comment by mrshenli** [comment]
Fixed by #20850
